apps/mull-api/src/app/entities/post.entity.ts

```diff
@@ -13,30 +14,38 @@ import { PostReaction } from './post-reaction.entity';
 import { User } from './user.entity';
 
 @Entity()
+@ObjectType()
 export class Post {
+  @Field(() => Int)
   @PrimaryGeneratedColumn()
   id: number;
 
-  @OneToOne(() => User)
-  @JoinColumn()
+  @Field(/* istanbul ignore next */ () => User)
+  @ManyToOne(/* istanbul ignore next */ () => User, /* istanbul ignore next */ (user) => user.posts)
   user: User;
 
+  @Field({ nullable: true })
   @OneToOne(() => Post)
   @JoinColumn()
-  parentPost: Post;
+  parentPost?: Post;
 
+  @Field()
   @Column()
   message: string;
 
+  @Field()
   @Column()
   createdTime: Date;
 
+  @Field(() => [Media], { nullable: true })
   @OneToMany(() => Media, (media) => media.post)
-  medias: Media[];
+  medias?: Media[];
 
+  @Field(() => [PostReaction], { nullable: true })
   @OneToMany(() => PostReaction, (reaction) => reaction.post)
-  reactions: PostReaction[];
+  reactions?: PostReaction[];
```

Comment:
⚠ Idk if these should be nullable since it could just return empty arrays when there's no media or reaction.

Did you run into TS issues, which made you put these as optional/nullable?

Reply:
It wouldn't let me create a post without creating media. Maybe I'm doing something wrong.

I think we should have a discussion about reactions.

Reply:
Though the channelInput is only nullable for this ticket. I will add a TODO

Reply:
I actually should not have set it as blocking, its minor imo, but just wanted to know more about the why behind this change. Yeah lets discuss